src/common.c

```diff
@@ -32,3 +32,13 @@ void adc_clear_flag(uint32_t adc_peripheral, uint32_t flag)
 {
 	ADC_SR(adc_peripheral) = ~flag;
 }
+
+/**
+ * @brief Return the sign of a number.
+ *
+ * @param number The number to get the sign from.
+ */
```

Comment:
Include @return on the header

Reply:
2bbcb634d0ad7f97cc146f69c3ce517b1819483b